src/installer.ts

```diff
@@ -170,6 +175,36 @@ export async function getNode(
   core.addPath(toolPath);
 }
 
+function isLts(versionSpec: string): boolean {
```

Comment:
nitpick: may be rename to `isLtsVersion`?

Reply:
✔️

Reply:
@maxim-lobanov Thinking again, maybe even `isLtsAlias`? Since it's actually checking whether it's an alias or not, instead of checking whether the version is actual LTS or not.

Reply:
✔️